src/components/App/App.tsx

```diff
@@ -1,25 +1,17 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import './App.scss';
+import { routes } from '../../routes/index';
+import { Route, Switch } from 'react-router-dom'
+// import Header from '../../components/Header/Header'
```

Comment:
This is something that we can remove going forward now that we are importing the components via routes

Reply:
Do you mean the `Header` @ErinUntermeyer? I actually think we'll want to leave it here so it can be rendered by `App` before we render any route beneath it.